build.sbt

```diff
@@ -37,12 +37,20 @@ lazy val core = project
   .settings(Testing.settings:_*)
 
 lazy val cluster = project
+  .dependsOn(core)
+  .aggregate(core)
   .settings(moduleName := "kanaloa-cluster")
   .configs(Testing.Integration)
   .settings(commonSettings:_*)
   .settings(Dependencies.settings:_*)
   .settings(Format.settings:_*)
   .settings(Publish.settings:_*)
   .settings(Testing.settings:_*)
+  .settings(ClusterTests.settings:_*)
+  .settings(
+    libraryDependencies ++= Dependencies.akkaCluster
+  ).configs(MultiJvm)
 
 
+addCommandAlias("validate", ";root;clean;compile;test;integration:test;multi-jvm:test")
```

Comment:
The "validate" command is a common practice to include all tasks that "validates" a build, for now we only have tests but we might add docs generation and benchmark in the future. I understand naming something based on the vision about it rather than what it does can be confusing for some cases, but here "validate" is a rather common convention, which would help avoid confusion.